### PR TITLE
Display calculator UI

### PR DESCRIPTION
**Pull request by yayoamigo**
Create a calculator React component
 Use class-based component

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (tufoinnkuo10) [inline src/App.js]: - [ ] Great job. please kindly ensure that the App.js component should be class-based

File: src/components/Buttons.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+// import React from 'react';
+
+// const Buttons = ({ color, sign }) => (
+//   <div className="flex">
+//     <button className="btn" style={{ backgroundColor: color }} type="button">{sign}</button>
+//   </div>
+// );
+
+// export default Buttons;
```
Comment (tufoinnkuo10) [inline src/components/Buttons.js]: - [ ] kindly remove commented code for clean code practice. nice work

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @yayoamigo 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅  components well used

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Buttons.js (hunk 2 above)
Comment (Whoistolu) [reply src/components/Buttons.js]: Well resolved 👍🏽

File: src/App.js (hunk 1 above)
Comment (Whoistolu) [reply src/App.js]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @yayoamigo 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**